labonneboite/common/search.py

```diff
@@ -441,7 +441,13 @@ def get_companies_from_es_and_db(json_body, sort):
             company_dict[obj.siret] = obj
 
         for siret in siret_list:
-            company = company_dict[siret]
+            try:
+                company = company_dict[siret]
+            except KeyError as e:
+                msg = "ES and DB out of sync: siret %s is in ES but not in DB - this should never happen" % siret
+                e.message = msg
```

Comment:
Ça ne serait pas plus court de faire directement : `e.message = "ES and DB out of sync: siret %s is in ES but not in DB - this should never happen" % siret` ?

Reply:
je pensais que la ligne serait trop longue pour pylint mais non, au temps pour moi ;)